packages/pigeon/pigeons/core_tests.dart

```diff
@@ -12,22 +12,45 @@ enum AnEnum {
 
 // A class containing all supported types.
 class AllTypes {
-  bool? aBool;
-  int? anInt;
-  double? aDouble;
-  String? aString;
-  Uint8List? aByteArray;
-  Int32List? a4ByteArray;
-  Int64List? a8ByteArray;
-  Float64List? aFloatArray;
+  AllTypes(
+    this.aBool,
+    this.anInt,
+    this.aDouble,
+    this.aString,
+    // TODO(stuartmorgan): Enable these once they work for all generators;
+    // currently at least Swift is broken.
+    // See https://github.com/flutter/flutter/issues/115906
+    // this.aByteArray,
+    // this.a4ByteArray,
+    // this.a8ByteArray,
+    // this.aFloatArray,
+    this.aList,
+    this.aMap,
+    this.nestedList,
+    this.mapWithAnnotations,
+    this.mapWithObject,
+    this.anEnum,
+  );
+
+  bool aBool;
+  int anInt;
+  double aDouble;
+  String aString;
+  // TODO(stuartmorgan): Enable these once they work for all generators;
+  // currently at least Swift is broken.
+  // See https://github.com/flutter/flutter/issues/115906
+  // Uint8List aByteArray;
+  // Int32List a4ByteArray;
+  // Int64List a8ByteArray;
+  // Float64List aFloatArray;
   // ignore: always_specify_types, strict_raw_type
-  List? aList;
+  List aList;
   // ignore: always_specify_types, strict_raw_type
-  Map? aMap;
-  List<List<bool?>?>? nestedList;
-  Map<String?, String?>? mapWithAnnotations;
-  Map<String?, Object?>? mapWithObject;
-  AnEnum? anEnum;
+  Map aMap;
+  List<List<bool?>?> nestedList;
+  Map<String?, String?> mapWithAnnotations;
+  Map<String?, Object?> mapWithObject;
```

Comment:
I would remove these three for now, since we don't currently support non-nullable generics and the only thing being specifically tested here is generics, so there's nothing conceptually being tested here that the AllNullableTypes version doesn't cover.